tests/unit/test_row_noise.py

```diff
@@ -1,11 +1,43 @@
+import numpy as np
+import pandas as pd
 import pytest
+from vivarium.framework.randomness import RandomnessStream
 
-from pseudopeople.noise_functions import omit_rows
+from pseudopeople.configuration import get_configuration
+from pseudopeople.noise_entities import NOISE_TYPES
 
+RANDOMNESS = RandomnessStream(
+    key="test_row_noise", clock=lambda: pd.Timestamp("2020-09-01"), seed=0
+)
 
-@pytest.mark.skip(reason="TODO")
-def test_omission():
-    pass
+
+@pytest.fixture(scope="module")
+def dummy_data():
```

Comment:
Don't we already have a fixture called `dummy_data`? Also, we're only using this fixture for the one test, so there's no need for it to have `scope="module"`

Reply:
I think there is a dummy_dataset in test_column noise.  We should also be able to use this for duplication noise function but I will remove it for now.